metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi-lexical-grammar.rule

```diff
@@ -431,6 +431,15 @@ optional_parameter: "optional" WS_INLINE parameter
 each_expression:    "each" WS_INLINE each_expression_body
 
 each_expression_body:   function_body
+                    |   each_expression_body
+                    |   invoke_expression NEWLINE? WS_INLINE? "+" NEWLINE? WS_INLINE? invoke_expression
```

Comment:
we already have `logical_or_expression`, `logical_and_expression`, `relational_expression`, etc in the grammar - can we use those instead of copying?

Reply:
I tried that but they are not working.

Reply:
Have you looked into why they're not working?

At minimum, please add a comment about why it was done this copy-paste way instead of using the other relational operation grammars

Reply:
added comments